python_benchmarks/core.py

```diff
@@ -24,34 +24,86 @@ def get_device_properties() -> Tuple[int, float]:
     # Device attribute enums (taken from cuda.h)
     # https://nvidia.github.io/cuda-python/module/cuda.html
```

Comment:
This links directly to the enum docs: https://docs.nvidia.com/cuda/cuda-driver-api/group__CUDA__TYPES.html#group__CUDA__TYPES_1ge12b8a782bebe21b1ac0091bf9f4e2a3